src/app/components/global-header/global-header.component.html

```diff
@@ -1,9 +1,16 @@
-<section class="vf-hero | vf-u-fullbleed vf-hero-hca" style="background-color: #3355aa">
+<section
+  class="vf-hero | vf-u-fullbleed vf-hero-hca"
+  style="background-color: #3355aa"
+>
   <div class="vf-hero__content | vf-box | vf-stack vf-stack--400">
-    <h2 class="vf-hero__heading"><a class="vf-hero__heading_link" routerLink="/"> {{ headerTitle }} </a></h2>
+    <h2 class="vf-hero__heading">
+      <a class="vf-hero__heading_link" routerLink="/"> {{ headerTitle }} </a>
+    </h2>
     <p class="vf-hero__subheading">
       {{ headerText }}
-      <a *ngIf="showAboutLink" class="vf-hero__link" routerLink="/about">About the catalogue</a>
+      <a *ngIf="showAboutLink" class="vf-hero__link" routerLink="/about"
+        >About the catalogue</a
+      >
```

Comment:
Did you say you have an automated tool adding these because I think we need to find another solution, I drastically prefer the line before the commit to the chance after.

Reply:
I totally agree. Yes, it's [prettier](https://prettier.io/). I like that it's formattign so we have a consistent style, I just don't like the style

Reply:
https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/362